i18n/english.yml

```diff
@@ -1300,6 +1301,17 @@ components:
     title: Timetable editor keyboard shortcuts
   TimezoneSelect:
     placeholder: Select timezone...
+  TransformationsViewer:
+    columnsAdded: "Custom columns added: %columns%"
+    noTransformationApplied: No transformations applied.
+    rowsAdded: "Rows added: %rows%"
+    rowsDeleted: "Rows deleted: %rows%"
+    rowsUpdated: "Rows updated: %rows%"
+    tableModified: Table Modified 
+    tableAdded: Table Added
+    tableReplaced: Table Replaced
+    tableDeleted: TableDeleted
```

Comment:
```suggestion
    tableDeleted: Table Deleted
```

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/950/commits/51e47ec5e0a32e1ee5752c287a72a9409c9a428b